Admin/AbstractAdminExtension.php

```diff
@@ -186,4 +186,12 @@ public function configureActionButtons(AdminInterface $admin, $list, $action, $o
     {
         return $list;
     }
+
+    /**
+     * {@inheritdoc}
```

Comment:
This will have no effect because the method is commented on the interface.

Please copy/paste it and add a `NEXT_MAJOR:` key to remember to remove it.